tests/reference/asr-test_import_02-55b47fa.stderr

```diff
@@ -1,4 +1,4 @@
-warning: The module 'test_import/test_import_2' cannot be loaded
+warning: The module 'test_import_2' located in tests/errors/test_import/test_import_2.py cannot be loaded
```

Comment:
This is a good change / test.

Reply:
Yes. I provided the full path to the file which is not being loaded so that its easy for us and the users to actually see what went wrong and where. Before this it was a bit cryptic.